tests/test_unittest/test_gtc/test_cuir_compilation.py

```diff
@@ -17,10 +17,7 @@
 import pytest
 import setuptools
 
-from gt4py import (  # TODO(havogt) this is a dependency from gtc tests to gt4py, ok?
-    config,
-    gt_src_manager,
-)
+from gt4py import config  # TODO(havogt) this is a dependency from gtc tests to gt4py, ok?
```

Comment:
It looks like this is only used for `config.GT_INCLUDE_PATH` later in a test. Do we still need that with a pip package?

Reply:
fixed.